setup.md

```diff
@@ -2,4 +2,54 @@
 title: Setup
 ---
 
-#FIXME
+## Setup Python
+
+Make sure that you have the latest Python installed. The latest Python released is `Python 3.11.3`
```

Comment:
This explicit version adds maintenance burden. Use fixed link to the latest version, e.g. https://www.python.org/downloads/.

But this is also incorrect. We should not expect that this training only works with the latest python version. We should include a requirement that at least python 3.11 is needed, if that is the case.